File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void) {
do {
   float dollars = get_float("Give value: ");
} 
while (dollars <  0);
   int cents = round(dollars) * 100;
   int coins = 0;
 do {
   coins++;
 } 
 while (cents >= 25);
 do {
coins++;
 }
 while (cents >= 10);
 do {
coins++;
 }
 while (cents >= 5);
 do {
coins++;
 }
 while (cents >= 1);
printf("%i\n", coins);
}

File: cash.py
from cs50 import get_float

cash = get_float("Change owed: ")

c = round(cash * 100)
change = 0
while c > 0:
    if c >= 25:
        c-= 25
        change+= 1
    elif c >= 10:
        c-= 10
        change+= 1
    elif c >= 5:
        c-= 5
        change+= 1
    elif c >= 1:
         c-= 1
         change+= 1
print(change)
